utils/Redirect-map.txt

```diff
@@ -202,7 +202,7 @@
 /basic-docs/atomicdex/atomicdex-tutorials/api-walkthrough.html https://komodoplatform.com/en/docs/komodo-defi-framework/tutorials/api-walkthrough/;
 /basic-docs/atomicdex/atomicdex-tutorials/coins-file-update.html https://komodoplatform.com/en/docs/komodo-defi-framework/tutorials/coins-file-update/;
 /basic-docs/atomicdex/atomicdex-tutorials/how-to-become-a-liquidity-provider.html https://komodoplatform.com/en/docs/komodo-defi-framework/tutorials/how-to-become-a-liquidity-provider/;
-/basic-docs/atomicdex/atomicdex-tutorials/how-to-compile-mm2-from-source.html https://komodoplatform.com/en/docs/komodo-defi-framework/tutorials/how-to-compile-mm2-from-source/;
+/basic-docs/atomicdex/atomicdex-tutorials/how-to-compile-kdf-from-source.html https://komodoplatform.com/en/docs/komodo-defi-framework/tutorials/how-to-compile-kdf-from-source/;
```

Comment:
/basic-docs/atomicdex/atomicdex-tutorials/how-to-compile-kdf-from-source.html revert to /basic-docs/atomicdex/atomicdex-tutorials/how-to-compile-mm2-from-source.html

Reply:
fixed in https://github.com/KomodoPlatform/komodo-docs-mdx/pull/364/commits/3a50e29fe9baa2c27359bbd88378f3cbcd13de24